app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+  validates :name, presence: true, length: { maximum: Settings.user.name }
+  VALID_EMAIL_REGEX = Settings.user.regex_email
+  validates :email, presence: true, length: { maximum: Settings.user.email },
+                    format: { with: VALID_EMAIL_REGEX },
```

Comment:
```suggestion
                    format: { with: Settings.user.regex_email },
```4
Truyền như này được không em

Reply:
Được ạ. Vì cùng gọi đến 1 giá trị